.github/workflows/sanity-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Sanity tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  # Run once per week (Monday at 04:00 UTC)
+  schedule:
+    - cron: '0 4 * * 1'
+
+jobs:
+  build:
+    name: '${{ matrix.name }}'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: latest Ansible 5 release
+            options: '-e galaxy_requirements="{{ playbook_dir | dirname }}/5/galaxy-requirements.yaml"'
+          - name: latest Ansible 6 release
+            options: '-e galaxy_requirements="{{ playbook_dir | dirname }}/6/galaxy-requirements.yaml"'
+
+    steps:
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v3
+        with:
+          path: ansible-build-data
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install ansible-core
```

Comment:
Hmm, this means both jobs run with the same version of ansible-core (and ansible-test) -- in this case 2.12.5, while ansible 6 is meant to be tested with 2.13.

It's not what I had intended, I guess versions of ansible-core should also be in the matrix.